docs/pages/eas/hosting/get-started.mdx

```diff
@@ -39,7 +39,7 @@ Paid subscribers can create more deployments, have more bandwidth, storage, requ
 
 <Collapsible summary="An Expo Router web project">
 
-If using an existing project, ensure that in your project's app config, the web output ([`expo.web.output`](/versions/latest/config/app/#output)) is set to either `static` or `server` (single page apps are not supported on EAS Hosting).
+If using an existing project, ensure that in your project's app config, the web output ([`expo.web.output`](/versions/latest/config/app/#output)) is set to either `static`, or `server` (single page apps are not supported on EAS Hosting).
```

Comment:
```suggestion
If using an existing project, ensure that in your project's app config, the web output ([`expo.web.output`](/versions/latest/config/app/#output)) is set to either `static`, or `server`.
```
Did you mean to remove this caveat?

Reply:
Nice catch! Didn't spot that one

Reply:
I'll remove this paragraph entirely, I think